build/linux/linux.spec

```diff
@@ -0,0 +1,100 @@
+# -*- mode: python ; coding: utf-8 -*-
+from PyInstaller.utils.hooks import collect_data_files
+from PyInstaller.utils.hooks import copy_metadata
+import os, torch, glob
+
+datas = []
+datas += collect_data_files('unidic_lite')
+datas += collect_data_files('manga_ocr')
+datas += copy_metadata('tqdm')
+datas += copy_metadata('regex')
+datas += copy_metadata('sacremoses')
+datas += copy_metadata('requests')
+datas += copy_metadata('packaging')
+datas += copy_metadata('filelock')
+datas += copy_metadata('numpy')
+datas += copy_metadata('tokenizers')
+
+added_files = [
+  ('../../app/assets', './assets'),
+  ('../../app/bin', './bin'),
+  ('../../app/components', './components')
```

Comment:
Without these I think Poricom gets an error message. I actually didn't know there is a function to open PDF since most of my files are in ZIP.
Also, unrar.exe is unlikely to work in Linux for opening rar files, but most people should have the unrar tool in their distro.  
Today I am mostly outside, will check this out within next week.